### PR TITLE
Daily Hunt Bid Adapter: update prebid compliance and add adapter back

### PR DESCRIPTION
**Pull request by shivam-pradhan**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: '<bidder name>',
  params: {
    // ...
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [ ] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

**Comment by ChrisHuie** [comment]
@shivam-pradhan can you please also submit documentation to our docs repo (https://github.com/prebid/prebid.github.io/pulls) for this new adapter?

File: modules/dailyhuntBidAdapter.js [hunk 1]
```diff
@@ -0,0 +1,435 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import * as mediaTypes from '../src/mediaTypes.js';
+import * as utils from '../src/utils.js';
```
Comment (ChrisHuie) [inline modules/dailyhuntBidAdapter.js:3]: can you please update these * imports to just pull in the needed functions? We recently went through the project to remove all the 'utils.' root from bid adapters.

Comment (shivam-pradhan) [reply modules/dailyhuntBidAdapter.js]: imported only used functions from utils.

**Comment by shivam-pradhan** [comment]
@ChrisHuie Also submitted the doc on docs repo (https://github.com/prebid/prebid.github.io/pulls)

**Comment by shivam-pradhan** [comment]
@ChrisHuie Please review.

**Comment by muuki88** [comment]
Old adapter https://github.com/prebid/Prebid.js/pull/4670

**Comment by montu1996** [comment]
@ChrisHuie 

Any update on review ?

**Review by ChrisHuie — APPROVED** [review]

**Comment by shivam-pradhan** [comment]
@ChrisHuie / @cpabst  / @onlsol  Just wondering if this pull request has been successfully merged, why dailyhunt adapter is not there in latest version.

**Comment by shivam-pradhan** [comment]
@eugene-krivenko I am not getting the checklist for dailyhunt adapter at https://docs.prebid.org/download.html. Please suggest I am missing something.

**Comment by shivam-pradhan** [comment]
@ChrisHuie @muuki88 @gwhigs @eugene-krivenko  I am not getting the checklist for dailyhunt adapter at https://docs.prebid.org/download.html. Please help if I am missing something.

**Pull request merged**